jupyterhub_traefik_proxy/proxy.py

```diff
@@ -47,15 +47,29 @@ class TraefikProxy(Proxy):
     )
 
     traefik_api_validate_cert = Bool(
-        True,
-        config=True,
-        help="""validate SSL certificate of traefik api endpoint""",
+        True, config=True, help="""validate SSL certificate of traefik api endpoint"""
     )
 
     traefik_log_level = Unicode("ERROR", config=True, help="""traefik's log level""")
 
+    traefik_https_port = Integer(8443, config=True, help="""https port""")
+
+    traefik_auto_https = Bool(
+        False, config=True, help="""enable automatic HTTPS with Let's Encrypt"""
+    )
+
+    traefik_letsencrypt_email = Unicode(config=True, help="""Let's Encrypt email""")
+
+    traefik_letsencrypt_domains = List(config=True, help="""domains list""")
+
+    traefik_acme_server = Unicode(config=True, help="""the CA server to use""")
```

Comment:
Does it make sense for the default acme server to be letsencrypt staging? `https://acme-staging-v02.api.letsencrypt.org/directory`

That way it works while you are testing without configuring the production server, plus we encourage folks to start using staging, which is a good idea anyway, since it avoids rate limit issues while folks redo initial deployments.